turbo.json

```diff
@@ -17,6 +17,7 @@
         "NEXTAUTH_ISSUER",
         "NEXTAUTH_URL",
         "NEXTAUTH_WELL_KNOWN_OVERRIDE",
+        "NEXTAUTH_AUTHORIZATION_SCOPE",
```

Comment:
Nice one, I would suggest to rename the env var to `NEXTAUTH_AUTHORIZATION_SCOPE_OVERRIDE`